src/lib/Components/ScrollableTabBar.tsx

```diff
@@ -26,19 +26,18 @@ const Tabs = styled.ScrollView`
   border-color: ${color("black10")};
 `
 
-const TabButton = styled(View)<{ active: boolean }>`
+const TabButton = styled(View)<{ active: boolean; isNonXPhone?: boolean }>`
   align-items: center;
   justify-content: center;
   height: 50px;
-  padding-left: 20px;
-  padding-right: 20px;
   flex-grow: 1;
   ${p =>
     p.active &&
     `
     border-color: ${color("black100")};
     border-bottom-width: 2px;
   `};
+  ${p => (p.isNonXPhone ? `padding-left:15px; padding-right:15px;` : `padding-left:25px; padding-right:20px;`)};
```

Comment:
This kind of 'is non-X iPhone' boolean tightly couples our `TabButton` component to the device it's running on (or, more specifically, whether or not its container is at least a certain width).

The bug really is that when you tap on a button that's partially offscreen, it doesn't get scrolled on screen. This PR solves that bug by making sure everything is on screen for this specific view, but a more long-term solution would be to modify the `ScrollableTabBar` component to scroll itself when a button gets tapped. Would you like to pair?